src/schema/nmdc.yaml

```diff
@@ -955,8 +955,13 @@ enums:
         aliases:
           - bottom_layer
       organic_layer:
```

Comment:
proposal to be more explicit. methanol, water, chloroform. If additional protocols with other types we can expand the PVs

protein - inter layer

Reply:
None of the existing PVs are populated in DB, removing now and getting more specific, then adding more will be a smaller lift, then using these incorrectly